tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_snapshot.py

```diff
@@ -132,11 +132,23 @@ def test_multi_target_autoload(remover):
     snapshot2 = extensions.snapshot_object(target, 'myfile.dat',
                                            autoload=True)
     # Load the snapshot and verify it
-    snapshot2.initialize(new_trainer)
+    assert snapshot2.initialize(new_trainer) == 'myfile.dat'
     assert new_trainer.state_dict() == trainer.state_dict()
     assert new_other.state_dict() == other_state_dict
 
 
+def test_multi_target_autoload_not_found(remover):
+    trainer = get_trainer_with_mock_updater(out_dir='.')
```

Comment:
The CI complains that this is not valid, but is actually valid and  runs well in local env!?

Reply:
rebased & pushed